Resources/Private/BackendModule/src/Components/ConfigurationForm.tsx

```diff
@@ -1,143 +1,52 @@
-import React, {ReactElement} from "react";
+import PureComponent from "./PureComponent";
 import {connect} from "react-redux";
-import PropTypes from "prop-types";
-import AssetModuleConfigurationInterface, {
-    AssetPropertyName, Language,
-    OnlyAssetsInUse
-} from "../Model/AssetModuleConfigurationInterface";
 import StateInterface from "../Store/StateInterface";
-import {setModuleConfiguration} from "../Store/AppSlice";
-import PureComponent from "./PureComponent";
-import {enumKeys} from "../Util";
+import React from "react";
+import AssetModuleConfigurationForm from "./AssetModuleConfigurationForm";
+import PropTypes from "prop-types";
+import FocusKeywordModuleConfigurationForm from "./FocusKeywordModuleConfigurationForm";
 
 @connect((state: StateInterface) => ({
     started: state.app.started,
-    configuration: state.app.moduleConfiguration,
+    scope: state.app.scope,
     hasError: state.app.hasError,
-    initialConfiguration: state.app.initialModuleConfiguration,
     backendMessage: state.app.backendMessage
```

Comment:
In der index holst du dir die Backend-Message, dann speicherst du es in den Store und hier holst du es wieder aus dem Store. 

Stattdessen fände ich es viel simpler zu verstehen, und leichter zum testen, wenn das alles in dieser einen Datei steht. 

Ungetestet stell ich mir das so vor:
```js
    componentDidMount() {
        const {moduleConfiguration, scope} = this.props;
        const externalService = ExternalService.getInstance();
        externalService.configure(moduleConfiguration.apiDomain, moduleConfiguration.apiKey, moduleConfiguration.userInterfaceLanguage)
        externalService.getBackendNotification(backendMessageIdentifier).then((data) => this.setState({backendMessage: data}));
    }
```